src/many-client/src/client/ledger.rs

```diff
@@ -0,0 +1,59 @@
+use many_identity::{Address, CoseKeyIdentity};
+use many_modules::ledger::{BalanceArgs, BalanceReturns, InfoReturns, SendArgs};
+use many_protocol::ManyError;
+pub use many_types::ledger::{Symbol, TokenAmount};
+use many_types::VecOrSingle;
+
+use crate::ManyClient;
+
+#[derive(Clone, Debug)]
+pub struct LedgerClient {
+    client: ManyClient,
+}
+
+impl LedgerClient {
+    pub fn new(client: ManyClient) -> Self {
+        LedgerClient { client }
+    }
+
+    pub async fn info(&self) -> Result<InfoReturns, ManyError> {
+        let response = self.client.call_("ledger.info", ()).await?;
+        minicbor::decode(&response).map_err(ManyError::deserialization_error)
+    }
+
+    pub async fn balance(
+        &self,
+        account: Option<Address>,
+        symbols: Option<Vec<Symbol>>,
+    ) -> Result<BalanceReturns, ManyError> {
+        let argument = BalanceArgs {
+            account,
+            symbols: symbols.map(VecOrSingle),
+        };
+        let data = self.client.call_("ledger.balance", argument).await?;
+        minicbor::decode(&data).map_err(ManyError::deserialization_error)
+    }
+
+    pub async fn send(
+        &self,
+        from: CoseKeyIdentity,
+        to: Address,
+        amount: TokenAmount,
+        symbol: Symbol,
+    ) -> Result<(), ManyError> {
+        let client = ManyClient::new(
+            self.client.url.clone(),
+            CoseKeyIdentity::anonymous().identity,
+            from.clone(),
+        )
+        .map_err(|_| ManyError::could_not_route_message())?;
```

Comment:
I don't believe this is needed if the original `ManyClient` is properly initialized.

Reply:
@fmorency the idea here is that on the specification tests I have a single ledger client initialized with the faucet private key, but I still want to send tokens from another private key. Do you recommend me to have one ledger client per private key instead, then?

Reply:
This is related to https://github.com/many-protocol/specification/pull/53#discussion_r954985507

If `to` is anonymous, then "the first server who receives this message should assume he is the destination."

See https://github.com/many-protocol/specification/blob/6ba25eebec3493340e6537682eb360ba24046042/spec/cddl/many.cddl#L12-L16

Reply:
I am suggesting to defer this configuration where the `ManyClient` is initialized.

Reply:
Your recent changes fix this.